packages/api/src/command/medical/admin/upload-doc.ts

```diff
@@ -119,6 +121,51 @@ export async function createAndUploadDocReference({
   });
 
   await fhirApi.updateResource(data);
-
   return data;
 }
+
+/**
+ * Fetches a DocumentReference draft from the FHIR servers and updates its status and file content information.
+ *
+ * @param cxId The ID of the organization uploading a document
+ * @param fileData The file metadata and DocumentReference ID
+ */
+export async function updateAndUploadDocumentReference({
```

Comment:
nit: this function name is a bit misleading, there's no upload here - that word when close to "document" leads to a certain understanding... 😄

Reply:
True. I think `update` for this function and `amend` for the one that adds the file location and changes the status etc